userspace/libsinsp/event.cpp

```diff
@@ -2179,10 +2180,14 @@ void sinsp_evt_param::throw_invalid_len_error(size_t requested_length) const
 
 	std::stringstream ss;
 	ss << "could not parse param " << m_idx << " (" << parinfo->name << ") for event "
-		<< m_evt->get_num() << " of type " << m_evt->get_type() << " (" << m_evt->get_name() << "): expected length "
-		<< requested_length << ", found " << m_len;
+		<< m_evt->get_num() << " of type " << m_evt->get_type() << " (" << m_evt->get_name() << "), for tid " << m_evt->get_tid() 
+		<< ": expected length " << requested_length << ", found " << m_len;
+	std::string error_string = ss.str();
 
-	throw sinsp_exception(ss.str());
+	libsinsp_logger()->log(error_string, sinsp_logger::SEV_DEBUG);
```

Comment:
since sinsp logs are disabled by default and here we are triggering an exception I think we could also use `FATAL` or `CRITICAL`

Reply:
ok, done!